scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -46,14 +47,27 @@ CHT_NETWORK=$projectName-cht-net
 EOL
 }
 
-get_latest_release() {
-  latest=$(curl -s https://staging.dev.medicmobile.org/_couch/builds_4/_design/builds/_view/releases\?limit\=1\&descending\=true |  tr -d \\n | grep -o 'medic\:medic\:[0-9\.]*')
-  echo "https://staging.dev.medicmobile.org/_couch/builds_4/${latest}"
+get_compose_download_url() {
+	preferredRelease=$1
+	if [ -z "$preferredRelease" ]; then
```

Comment:
I think you've got a mix of tabs and spaces which is leading to indentation issues here.

Reply:
ah - sorry about that!  fixed all tabs -> spaces in 2d5fff2c8